frontend/src/features/admin-form/settings/components/FormIssueNotificationToggle.tsx

```diff
@@ -0,0 +1,38 @@
+import { useCallback, useMemo } from 'react'
+import { Skeleton } from '@chakra-ui/react'
+
+import Toggle from '~components/Toggle'
+
+import { useMutateFormSettings } from '../mutations'
+import { useAdminFormSettings } from '../queries'
+
+export const FormIssueNotificationToggle = (): JSX.Element => {
+  const { data: settings, isLoading: isLoadingSettings } =
+    useAdminFormSettings()
+
+  const hasIssueNotification = useMemo(
+    () => settings && settings?.hasIssueNotification,
+    [settings],
+  )
+
+  const { mutateFormIssueNotification } = useMutateFormSettings()
+
+  const handleToggleIssueNotification = useCallback(() => {
+    if (!settings || isLoadingSettings || mutateFormIssueNotification.isLoading)
+      return
+    const nextHasIssueNotification = !settings.hasIssueNotification
+    return mutateFormIssueNotification.mutate(nextHasIssueNotification)
+  }, [isLoadingSettings, mutateFormIssueNotification, settings])
+
+  return (
+    <Skeleton isLoaded={!isLoadingSettings && !!settings}>
```

Comment:
Do we need to check both of these? Is there a situation where the `isLoadingSettings` state is false but `settings` is undefined?

Reply:
Hmm goodpoint, I think it is unlikely, its improbable to get an empty setting after the query is successful after because of our form defaults anyway.

So if it succeeds, should have finish loaded, and setting should be defined.

Will remove the `!!settings` check!

Reply:
[13c69e3](https://github.com/opengovsg/FormSG/pull/6590/commits/13c69e3903770679ec2fb5e0f2adbdf38ae6412a)

Reply:
Oh there's another part of the code with the repeated checks too! Wait ah